.travis.yml

```diff
@@ -1,6 +1,36 @@
+sudo: false
+
 language: node_js
+
 node_js:
-  - "0.10"
+  - "4"
+  - "5"
+  - "6"
+
+cache:
+  directories:
+    - node_modules
+
+install:
+  - npm install
+
+script:
+  - npm run test
+  - npm run lint
+  - npm run build
```

Comment:
This two commands are not supported in `package.json`. Please, only run `npm test`